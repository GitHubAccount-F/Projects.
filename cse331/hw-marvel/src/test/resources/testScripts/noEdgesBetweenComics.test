#Tests when there is multiple characters in a comic,
#but there isn't an edge connecting to characters outside the
#comic.

#Creates empty graph
LoadGraph graph1 onlyTwoCharactersPerComic.csv

#Checks to make sure all Marvel characters were added
ListNodes graph1

#Tests if it can find a path between characters
#in the same comic(it should)
FindPath graph1 SpiderMan Venom

FindPath graph1 Venom SpiderMan

FindPath graph1 Thor Hulk

FindPath graph1 HawkEye QuickSilver

#Tests if can find a path between characters in
#different comics(it shouldn't)
FindPath graph1 SpiderMan Thor

FindPath graph1 Hulk HawkEye