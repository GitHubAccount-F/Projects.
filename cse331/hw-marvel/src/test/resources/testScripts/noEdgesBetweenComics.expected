#Tests when there is multiple characters in a comic,
#but there isn't an edge connecting to characters outside the
#comic.

#Creates empty graph
loaded graph graph1

#Checks to make sure all Marvel characters were added
graph1 contains: Captain-America HawkEye Hulk Iron-Man QuickSilver SpiderMan Thor Venom

#Tests if it can find a path between characters
#in the same comic(it should)
path from SpiderMan to Venom:
SpiderMan to Venom via Spider-Verse

path from Venom to SpiderMan:
Venom to SpiderMan via Spider-Verse

path from Thor to Hulk:
Thor to Hulk via Ragnarok

path from HawkEye to QuickSilver:
HawkEye to QuickSilver via Age-Of-Ultron

#Tests if can find a path between characters in
#different comics(it shouldn't)
path from SpiderMan to Thor:
no path found

path from Hulk to HawkEye:
no path found